recipes/quip/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "quip" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libAtoms/QUIP/archive/public.tar.gz
+  sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - make
+  host:
+    - python
+    - numpy
```

Comment:
If you are linking against numpy: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy